VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -196,7 +193,9 @@
                         android:layout_height="@dimen/course_unit_nav_bar_btn_height"
                         android:background="@color/white"
                         android:text="@string/assessment_previous"
-                        android:textColor="?attr/buttonTextColor" />
+                        android:textColor="?attr/buttonTextColor"
+                        android:gravity="center_vertical|start"
```

Comment:
Again, this gravity seems useless. Horizontal gravity does not apply as it's layout width is `wrap_content`, and vertical gravity is already centered for buttons I think.